src/lib/cj/client.rs

```diff
@@ -23,9 +25,17 @@ impl CJS2SClient {
         }
     }
 
+    fn random_minutes(&self) -> Duration {
+        let mut rng = thread_rng();
+        let minutes = rng.gen_range(15..=60);
+        Duration::minutes(minutes)
+    }
+
     pub async fn report_subscription(&self, sub: &Subscription) -> Result<Response, Error> {
         let mut url_for_sub = self.url.clone();
-        let event_time = sub.subscription_created.format("%FT%H:00:00.000Z");
+        // Note this must be in the future or will fail CJ side
+        let randomized_event_time = sub.subscription_created + self.random_minutes();
```

Comment:
Would be useful to know why we chose a randomized time here instead of just tacking on a static value to each event?

Reply:
Ok, reading the second PR it looks like this was done for privacy considerations, is that right?